cmd/vic-machine/create/create.go

```diff
@@ -876,14 +876,25 @@ func (c *Create) processVolumeStores() error {
 	defer trace.End(trace.Begin(""))
 	c.VolumeLocations = make(map[string]string)
 	for _, arg := range c.volumeStores {
-		if err := common.CheckUnsupportedCharsDatastore(arg); err != nil {
-			return fmt.Errorf("--volume-store contains unsupported characters: %s Allowed characters are alphanumeric, space and symbols - _ ( ) / :", err)
+		splitMeta := strings.Split(arg, ":")
+		if len(splitMeta) < 1 {
```

Comment:
There are two checks. One is in the validator and one is at parameter checking time. They are subtly different. The issue is that there is the possibility to reject some strings that we might want during this time. So we need a way to not reject those now since they are valid.